packages/airnode-node/src/constants.ts

```diff
@@ -1,8 +1,8 @@
 // The maximum TOTAL time an API call has before it is timed out (including retries).
-export const API_CALL_TOTAL_TIMEOUT = 10_000;
+export const API_CALL_TOTAL_TIMEOUT = 30_000;
 
 // The maximum time an API call has before it is timed out (and retried).
-export const API_CALL_TIMEOUT = 10_000;
+export const API_CALL_TIMEOUT = 30_000;
```

Comment:
I didn't realise that these times were the same (`API_CALL_TIMEOUT` and `API_CALL_TOTAL_TIMEOUT`). The original idea was that the worker itself gets (for example) 30 seconds to run, while the call gets less time - both per call and in total, as the name suggests. That way, if the call hangs for whatever reason, it will at least get retried once.

Now, it looks like we just execute a single API call and hope it succeeds? Not sure how big of a deal this is, just thought I would point it out

Reply:
Hey
As I understand it some APIs could take a long time to respond, so I thought making them the same and a high-ish value would serve the greatest selection of potential APIs. Worst case the API gets re-called a minute later, and assuming the first failure was due to transient network events, succeeds and results in a fulfilment.

Reply:
Yeah I figured that's probably the better option. We would prefer to support APIs that have very slow response times over a second (retried) API call in the same cycle.